apiary.apib

```diff
@@ -3055,6 +3057,8 @@ Get details for a single project.
                 + `provided` (Money)
                 + `spent` (Money)
                 + `remaining` (Money) - The actual budget remaining, not taking the planned work into account.
+                + `allocated` (Money, optional) - The estimated budget for planned work that is still remaining.
+                + `projected` (Money, optional) - The estimated budget at completion.
```

Comment:
Q: Are `allocated` and `projected` really optional? Why are they different to `provided`, `spent` and `remaining`?

Reply:
> Q: Are `allocated` and `projected` really optional? Why are they different to `provided`, `spent` and `remaining`?

First 3 are always specified, allocated and projected may or may not exist yet (`null`)

Reply:
But that's should be marked with `nullable`, not `optional`, no? They will always be there, but they may be null.